### DISCUSSION TITLE
Support PASERK IDs

### DISCUSSION THREAD

**Discussion by Eh2406** [panva/paseto#28] [Ideas]
The standard recommends using [PASERK IDs](https://github.com/paseto-standard/paserk/blob/master/operations/ID.md) for key ids. Does this library provide a way to convert a publicKey into a id? Would you be interested in adding the functionality?

**Comment by Extarys** [comment]
I was looking into a way to generate an ID that I can use in the footer of my token and discovered `lid`. So far I didn't find any node/js implementation of this.
